fairworkflows/rdf_wrapper.py

```diff
@@ -1,3 +1,4 @@
+import copy
```

Comment:
Is copy still needed now that pyshacl no longer mutates the graph?

Reply:
Nope, I removed it!